pkg/dyninst/integration_test.go

```diff
@@ -281,6 +287,8 @@ func testDyninst(
 		var decodeOut bytes.Buffer
 		probe, err := decoder.Decode(event, cachingSymbolicator, &decodeOut)
 		require.NoError(t, err)
+
+		t.Log("Output:", decodeOut.String())
```

Comment:
How would you feel about gating this on an environment variable? Or using `log.Tracef` so that `DD_LOG_LEVEL` will control it? I find it to be very noisy when just running the tests.

Reply:
I'll change it to log.Tracef!